client/cmddata.c

```diff
@@ -919,7 +919,9 @@ int CmdDetectClockRate(const char *Cmd)
 
 char *GetFSKType(uint8_t fchigh, uint8_t fclow, uint8_t invert)
 {
-	char *fskType;
+	static char fType[8];
```

Comment:
If it is not static the memory is released as soon as the function ends so the returned pointer is invalid.  

Also I'm Following the examples in the util.c